DMX.Gatekeeper.Api.Tests.Unit/Services/Foundations/LabCommands/LabCommandServiceTests.Logic.Modify.cs

```diff
@@ -0,0 +1,46 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ---------------------------------------------------------------
+
+using System.Threading.Tasks;
+using DMX.Gatekeeper.Api.Models.LabCommands;
+using FluentAssertions;
+using Force.DeepCloner;
+using Moq;
+using Xunit;
+
+namespace DMX.Gatekeeper.Api.Tests.Unit.Services.Foundations.LabCommands
+{
+    public partial class LabCommandServiceTests
+    {
+        [Fact]
+        public async Task ShouldModifyLabCommandAsync()
+        {
+            // given
+            LabCommand randomLabCommand = CreateRandomLabCommand();
+            LabCommand inputLabCommand = randomLabCommand;
+            LabCommand postedLabCommand = inputLabCommand;

```

Comment:
Did you mean `updatedLab` - copy pasta?

Reply:
![image](https://user-images.githubusercontent.com/1453985/186030937-266010b8-0078-45eb-a755-a576b74b3e11.png)